perfkitbenchmarker/benchmark_spec.py

```diff
@@ -103,11 +111,19 @@
         NETWORK: digitalocean_network.DigitalOceanNetwork,
         FIREWALL: digitalocean_network.DigitalOceanFirewall
     },
+    RACKSPACE: {
+        VIRTUAL_MACHINE: {
+            DEBIAN: rax_vm.DebianBasedRackspaceVirtualMachine,
+            RHEL: rax_vm.RhelBasedRackspaceVirtualMachine
+        },
+        NETWORK: rax_net.RackspaceNetwork,
```

Comment:
The commit https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/commit/c7233241ce41c9e629a9046ce2b40d4288ad7896 (see previous comment) also moved the NETWORK initialization. Since your network class does nothing, you should be able to delete the RackspaceNetwork class and just keep the firewall class.